wpl_cli/src/asset/instructions/create.rs

```diff
@@ -0,0 +1,102 @@
+use anyhow::Result;
+
+use clap::Parser;
+use solana_program::system_program::ID as SYSTEM_PROGRAM_ID;
+use solana_sdk::{
+    message::{v0::Message as TransactionMessage, VersionedMessage},
+    pubkey::Pubkey,
+    signature::Keypair,
+    signer::Signer,
+    transaction::VersionedTransaction,
+};
+use spl_associated_token_account::{
+    get_associated_token_address_with_program_id, ID as ASSOCIATED_TOKEN_PROGRAM_ID,
+};
+use spl_token_2022::ID as TOKEN_PROGRAM_ID;
+use wen_new_standard::{
+    instructions::{CreateMintAccount, CreateMintAccountInstructionArgs},
+    types::CreateMintAccountArgs,
+};
+
+use crate::{utils::derive_manager_account, Context};
+
+#[derive(Debug, Parser, Clone)]
+pub struct CreateArgs {
+    /// Name of the asset
+    #[arg(short, long)]
+    pub name: String,
+    /// Symbol of the asset
+    #[arg(short, long)]
+    pub symbol: String,
+    /// URI of the asset
+    #[arg(short, long)]
+    pub uri: String,
+    /// Receiver address of the asset
+    #[arg(short = 'R', long, value_parser = clap::value_parser!(Pubkey))]
+    pub receiver: Option<Pubkey>,
+    /// Optional permanent delegate
+    #[arg(short, long, value_parser = clap::value_parser!(Pubkey))]
+    pub permanent_delegate: Option<Pubkey>,
+}
+
+pub async fn run(context: Context, args: CreateArgs) -> Result<()> {
+    let Context { client, keypair } = context;
+    let payer = keypair.pubkey();
+    let recent_blockhash = client.get_latest_blockhash().await?;
+
+    let mint_keypair = Keypair::new();
```

Comment:
Whenever I see keypairs being generated from the Cli I wonder if people can supply their own mint keypair to grind custom addresses

Reply:
Will add this in the upcoming PR (asset uploads)